base_kanban_stage_state/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
Switch to LGPL-3. 

Future reference, anything our LasLabs/SMD teams create should always be LGPL-3 unless we are depending on an AGPL-3 module.